jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/common/Snackbar.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.jdi.light.angular.elements.common;
+
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.angular.asserts.SnackbarAssert;
+
+import com.epam.jdi.light.common.JDIAction;
+
+/**
+ * Angular snack-bar documentation: https://jdi-docs.github.io/jdi-light/?java#progress-spinner
```

Comment:
The link in this comment does not refer to a snack-bar but to a progress-spinner. This comment should only be added when the documentation is complete. I think this should be removed for now.

Reply:
fixed